identity-apps-core/components/org.wso2.identity.apps.common/src/main/java/org/wso2/identity/apps/common/util/AppPortalUtils.java

```diff
@@ -469,6 +485,58 @@ public static String resolveClientID(String consumerKey, String tenantDomain) {
             return consumerKey + "_" + tenantDomain;
         }
     }
+    
+    private static void addOImpersonatorRole(String appOwner, String appId, int tenantId, String tenantDomain)
```

Comment:
add**O**ImpersonatorRole is this intended?

Reply:
Address e89e418b2d300c603e25516ed724dc68092fd3f0